tests/e2e-tests/specs/admin/shipping-settings.test.js

```diff
@@ -0,0 +1,348 @@
+        const secondCardExpiryDate = await (await cardExpiryDates[MASTERCARD_INDEX].getProperty('innerText')).jsonValue();
+        expect(secondCardExpiryDate).toEqual('Expires 2025-12-31');
+
+        // Verify that no default box is checked.
+        const cardDefaultCheckbox = await page.$$('.label-settings__card-checkbox.form-checkbox');
+        const firstCardDefaultCheckbox = await (await cardDefaultCheckbox[VISA_CARD_INDEX].getProperty('checked')).jsonValue();
+        expect(firstCardDefaultCheckbox).toBeFalsy();
+        const secondCardDefaultCheckbox = await (await cardDefaultCheckbox[MASTERCARD_INDEX].getProperty('checked')).jsonValue();
+        expect(secondCardDefaultCheckbox).toBeFalsy();
+    });
+
+    it ('Should show the correct email receipts message', async () => {
+        // This test continue to use the mocked account/settings from the above test, which relies on AccountWithTwoCreditCardAndNoDefault.
+        await expect(page).toMatchElement('.label-settings__credit-card-description', { text: "Email the label purchase receipts to johndoe (johndoe) at john.doe@automattic.com" });
+    });
+
+    /**
+     * Not really a test case. But needed to run this once, after all test case ran in this describe.
+     */
+    it('should turn off request interception after all credit card test ran', async () => {
+        await page.setRequestInterception(false);
+    });
+});
+
+describe( 'Packaging', () => {
+    let metricSystemValue = ''; // either "in" or "cm".
+
+    it( 'Can add package' , async () => {
+        const packageName = 'Package Box 5x5x5';
+
+		await StoreOwnerFlow.login();
+        await StoreOwnerFlow.openSettings('shipping', 'woocommerce-services-settings');
+
+        // Wait for "Add package" to finish loading, click to pop up modal
+        await waitForSelectorAndText('.button:not([disabled])', 'Add package');
+        await expect( page ).toClick( '.button', { text: 'Add package' } );
+
+        // Create a new package
+        await waitForSelectorAndText( '.packages__add-edit-title.form-section-heading', 'Add a package' );
+        await expect( page ).toFill( '.packages__properties-group #name', packageName );
+        await expect( page ).toFill( '.form-text-input.form-dimensions-input__length', '5' );
+        await expect( page ).toFill( '.form-text-input.form-dimensions-input__width', '5' );
+        await expect( page ).toFill( '.form-text-input.form-dimensions-input__height', '5' );
+        await expect( page ).toFill( '.form-text-input-with-affixes #box_weight', '0.5' );
+        await expect( page ).toClick( '.button.form-button.is-primary', { text: 'Add package' } )
+
+        // Set this once globally, all following test will use this metric
+        const metricSystem = await page.$$('.form-text-input-with-affixes .form-text-input-with-affixes__suffix');
+        metricSystemValue = await (await metricSystem[0].getProperty('innerText')).jsonValue();
```

Comment:
The unit is either in `in` or `cm`. We need to get this from the system the tests are running on. 

We will find this unit the first time we run the tests, then save this in a variable and used it for all remaining tests.